src/server/config/sentry.ts

```diff
@@ -1,5 +1,5 @@
 const sentryDsn = process.env.SENTRY_DSN
-const sentryEnvironment = process.env.SENTRY_ENVIRONMENT || 'dev'
```

Comment:
Even if your env is set, does it fallback?
Don't we want to keep the fallback if we don't have it set?

Reply:
Yep, the env variable is just ignored if we have the fallback. Since it takes longer to get the env variable I guess - it's undefined for a fraction of a moment so then `sentryEnvironment` is set to `'dev'` and never change, at least locally. I mean, it works in production.
And correct me if I'm wrong - I don't think we need the fallback since the error would be logged to Sentry anyway - it will just not have a specified environment. And that is a more correct IMO than if it's just set to "dev" even if the environment isn't actually "dev".

Reply:
Nice! 👌